src/jit/emitinl.h

```diff
@@ -388,6 +421,35 @@ ssize_t             emitter::emitGetInsAmdAny(instrDesc *id)
                              regmask |= RBM_R11;
 #endif
 
+#ifdef _TARGET_ARM64_
+    assert(REGNUM_BITS >= 5);
+    encodeMask = id->idReg1();
+
+    if (encodeMask & 0x01)
+        regmask |= RBM_R19;
+    if (encodeMask & 0x02)
+        regmask |= RBM_R20;
+    if (encodeMask & 0x04)
+        regmask |= RBM_R21;
+    if (encodeMask & 0x08)
+        regmask |= RBM_R22;
+    if (encodeMask & 0x10)
+        regmask |= RBM_R23;
+
+    encodeMask = id->idReg2();
+
+    if (encodeMask & 0x01)
+        regmask |= RBM_R24;
+    if (encodeMask & 0x02)
+        regmask |= RBM_R25;
+    if (encodeMask & 0x04)
+        regmask |= RBM_R26;
+    if (encodeMask & 0x08)
+        regmask |= RBM_R27;
+    if (encodeMask & 0x10)
+        regmask |= RBM_R28;
+#endif
```

Comment:
I'm not sure what is the value of having them explicitly listed (and duplicated) - just because the existing code is poorly factored. I was thinking that it would be good to have something along the lines of DecodeRegMasks(id). You could then and it with the register mask of interest (i.e. RBM_CALLEE_SAVED)  It would probably be useful to have the inverse as well, but that isn't really relevant to this change.